docs: update pasta options description (#25769)
containers/podman: docs/source/markdown/options/network.md

    container to host using the gateway address. The latter can be overridden
    by passing **--map-gw** in the pasta-specific options (despite not being an
    actual pasta(1) option). \

>     For better integration with DNS handling **--dns-forward 169.254.1.1** is passed

Reviewer: ```suggestion
    For better integration with DNS handling, **--dns-forward 169.254.1.1** is passed,
```

>     and the ip is added to resolv.conf as first entry. It is possible to pass **--dns-forward**

Reviewer 2: and this address is added to resolv.conf(5) as first resolver.

>     explicitly in case a different ip should be used.

Reviewer: ```suggestion
    explicitly in case a different IP should be used.
```

Reviewer 2: ...IP address, even. IP is a protocol.

    To make the `host.containers.internal` /etc/hosts entry work and allow connections

>     to the host **--map-guest-addr 169.254.1.2** is passed as well, again it can be set

Reviewer: ```suggestion
    to the host, **--map-guest-addr 169.254.1.2** can also be passed. Again, it can be set
```

Author: It is not "can", it is set by default. It can also be passed by a user to overwrite it but podman will always set this as default option. I changed to just "is passed" like in the first sentence.

>     explicitly to choose a different ip. \

Reviewer: ```suggestion
    explicitly to choose a different IP. \
```

    Also, **-t none** and **-u none** are passed if, respectively, no TCP or

>     UDP port forwarding from host to container is configured (i.e. via Podman's

Reviewer 2: I would drop "i.e.", that's a pretty direct relationship between those options and the configuration of port forwarding.

>     **--publish** or by passing the pasta **-t/-u** options directly as options),

Reviewer 2: `**-t**/**-u**`

Reviewer 2: ...options directly),

>     **--publish** or by passing the pasta **-t**/**-u** options directly as options),

Reviewer 2: `passing pasta's **-t**/**-u** options directly),`

...but not a strong preference, I see why might want to specify "options as options", even though it feels redundant and not necessarily clearer to me.

Author: dropped

    to disable automatic port forwarding based on bound ports. Similarly, **-T none**
    and **-U none** are given to disable the same functionality from container to
    host. \

>     All options can also be set in **[containers.conf(5)](https://github.com/containers/common/blob/main/docs/containers.conf.5.md)**,

Reviewer: ```suggestion
    All options can also be set in **[containers.conf(5)](https://github.com/containers/common/blob/main/docs/containers.conf.5.md)**;
```

>     see `pasta_options` under the network section there. \

Reviewer: ```suggestion
    see the `pasta_options` under the network section in that file. \
```
